### ISSUE TITLE
[Feature] Issues with beta version 201005

### ISSUE THREAD

**Issue by yo1995** [DukeMobileDevCenter/Jigsaw#96]
- [x] One big question to discuss is how much of the content you can fill in in the next few days. We cannot  launch the app for beta testing as long as there are rooms with no content (quotations and quizzes).

- [x] BEFORE WE START
“Teams will be ranked on how quickly they are able to move through all of the rooms in this series.”
Should instead be:
“Teams will be ranked on the percentage of quiz questions that the team gets correct. Ties will be broken by how quickly the team ws able to move through all of the rooms in this series.”
    - ❓ do we do individual timing for each team or each player?

---

## Logistics

GAMES 2P
- [x] remove random as an option in list of topics
- [x] ❓  Grey out topics already done by this player. 
    - Games allow replaying the played levels. 

ECONOMY
- [x] Move unlock button so that it does not cover door (for all topic icons)

I – ECONOMY
- [x] CHANGE: “(Biden’s/Trump’s)” to “Biden’s or Trump’s”
- [x] ADD: Your teammate will see information about the other candidate. — BEFORE “Afterwards, …”
- [x] INSTEAD OF [SIC] put “…prov[id]ing” or just “…providing”
- [x] Edit obvious grammar and spelling errors throughout all quotations

- [x] DISABLE cut and paste in Chat so that players cannot just cut the quotation when they read it and paste it into the chat for others to see
    - related to #89 

REQUIRED QUESTIONS
- [x] Instead of “Questions” use label “Quiz”
- [x] Instead of “The following questions are essential. Please answer carefully.” Say “You can escape this room if your team answers all of the following questions correctly. Good luck!”

- [x] ❗  NO Queuing needed when same team enters second room.  - This need to be done by calling the IBAction.

- [x] ??? HANNAH, should we have the quotation visible in the chat? It is hard to remember. But they might just retype it. Ric thinks it is better not to allow this

- [x] ❓  PLAYERS SHOULD BE ABLE TO REVIEW THE QUOTATION (the one they saw before) AFTER THEY MISS A QUESTION AND HAVE TO GO BACK TO THE CHAT.
    - What if some players passed while some don't? Should everyone review the quotation materials?

- [x] Need to tell players whether they can choose more than one answer to each question.
    - can add prompt to description (is a multiple choice question)

- [ ] DON’T NEED GAME RESULT IF IT IS ALWAYS 100% BECAUSE YOU EACH MUST GET BOTH ANSWERS CORRECT. BUT maybe tell what percentage they got correct so far.

- [x] ADD ENCOURAGING WORDS LIKE “CONGRATULATIONS” EVERY TIME THEY ESCAPE A ROOM AND IN BIG PRINT when they finish all of the rooms on a topic.

- [x] CHANGE WORDING OF “END TASK” to what???

- [x] ❓  Should each player alternate between Biden and Trump? That seemed to happen.
    - currently the topics are randomly assigned. do you want to always give Biden to conserv/Trump to liberal?

- [x] Chat needs some button to show other team members that you are done and have no more info. Simpler than typing “Done”
    - Either send an auto message when a player is done, say "I'm done, bye!"
    - Or a caption text says "purple piece has left the chatroom". This will need another 1.5 hours to implement

---

## Bugs

- [x] AFTER A WHILE, IT SAID “120 SECONDS REMAINING” BUT THERE WAS NO TIMER ON THE SCREEN.
- [ ] PLAYERS SHOULD NOT LOSE THEIR PLACE WHEN YOUR SCREEN CLOSES FOR LACK OF ACTIVITY. I think this happened to me.
- [x] ALSO NO BUTTON AT TOP TO QUIT CHAT.
- [x] AT ONE TIME RIC WAS IN LEVEL 3 AND I WAS IN LEVEL 4, BUT WE WERE STILL IN THE SAME CHAT WINDOW. BUG?
    - it is a bug related to a previous unexpected exit.
- [x] GOT STUCK IN CHAT. THEN COLOR OF PIECE DID NOT DISPLAY IN THE CHAT.

**Comment by yo1995** [comment]
- [x] - The text for my Trump sentence was cut off/illegible. 
- [x] - I was kicked out of the chat despite having more time on the timer
- [x] - The timer went into negative time when we were bumped back having failed a quiz
- [x] - I didn't notice a screen for instructions prior to the chat -- should we add this (e.g. "Chat with your teammate about the different information that each of you just saw")? 
- [x] - We were kicked out of the game (i.e., "Jigsaw broken, etc.") perhaps too quickly (after failing twice) -- should we increase the number of tries a team gets before having to restart entirely?

**Comment by yo1995** [comment]
At the end of the game, provide a link to the feedback survey

what do you like about the game
what do you don't like about the game

contact information
